src/main/java/com/pretz/dyntreeserver/controller/AuthController.java

```diff
@@ -29,6 +29,7 @@ public AuthController(AuthUserService authUserService) {
      * @param userDTO - UserDTO containing info about user about to login.
      * @return - ResponseEntity containing user token if successful.
      */
+    //TODO Here JSON with token and status could be sent instead, check here: http://www.baeldung.com/java-json-web-tokens-jjwt (chapter 4: Building JWTs with JJWT)
```

Comment:
I'd propose to just set a session cookie

Reply:
why

Reply:
I don't remember. We had some discussion about this approach under one of previous pull requests:
https://github.com/Elm0xz/dyn-tree-server/pull/1#discussion_r161211871
Then we've chosen the JWT option and I already included logic of creating them. Parameters like session time can be included in token.

Reply:
I will create another task for research of cookie in context of JWT and taking into account an additional fact that I want to create a mobile app too.